src/ba-sak/formateringer.ts

```diff
@@ -24,7 +24,8 @@ export const formaterValgfelt = (valgfeltBlock: ValgfeltBlock, data: Begrunnelse
     case Valgfelttype.FOR_BARN_FØDT:
       return valgfeltSerializer(valgfeltBlock, hentForBarnFodtValg(data), data);
     case Valgfelttype.DU_OG_ELLER_BARNET_BARNA:
-      return valgfeltSerializer(valgfeltBlock, hentDuOgEllerBarnetBarnaValg(data), data);
+    case Valgfelttype.DEG_OG_ELLER_BARNET_BARNA:
```

Comment:
De fleste av disse valgene som allerede finnes følger dette mønsteret:

`**hent** Valgfelttype**valg**`, bortsett fra denne. 


Bør den byttes fra `Valgfelttype.DEG_OG_ELLER_BARNET_BARNA` til `Valgfelttype.SØKER_OG_ELLER_BARNET_BARNA` ?

Reply:
Scratch that, så ikke det var multiple cases 😛